src/main/java/uk/gov/hmcts/reform/ccd/document/am/controller/endpoints/CaseDocumentAmController.java

```diff
@@ -1,211 +1,182 @@
-        if (documentManagementService.checkUserPermission(responseEntity, documentId, authorization, Permission.UPDATE)) {
+        if (documentManagementService.checkUserPermission(responseEntity, documentId, Permission.UPDATE)) {
             return  documentManagementService.deleteDocument(documentId, userId, userRoles, permanent);
 
         }
-        LOG.error("User doesn't have update permission on requested document {}",  HttpStatus.FORBIDDEN);
+        LOG.error("User doesn't have update permission on requested document " + HttpStatus.FORBIDDEN);
         throw new ForbiddenException(documentId.toString());
     }
 
-    @Override
-    public ResponseEntity<Object> getDocumentBinaryContentbyDocumentId(
-        @ApiParam(value = "Service Auth (S2S). Use it when accessing the API on App Tier level.", required = true)
-        @RequestHeader(value = "ServiceAuthorization", required = true) String serviceAuthorization,
-        @ApiParam("documentId")
-        @PathVariable("documentId") UUID documentId,
-        @ApiParam("Authorization header of the currently authenticated user")
-        @RequestHeader(value = "Authorization", required = true) String authorization,
-        @ApiParam("User-Id of the currently authenticated user. If provided will be used to populate the creator field of a document"
-            + " and will be used for authorisation.")
-        @RequestHeader(value = "user-id", required = false) String userId,
 
-        @ApiParam("Comma-separated list of roles of the currently authenticated user. If provided will be used for authorisation.")
-        @RequestHeader(value = "user-roles", required = false) String userRoles) {
+    //**************** Binary content API ***********
+    @ApiOperation(value = "Streams contents of the most recent Document Content Version associated with the Case Document."
+         )
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Returns contents of a file", response = Object.class),
+       })
+    @RequestMapping(value = "/cases/documents/{documentId}/binary", produces = {APPLICATION_JSON}, method = RequestMethod.GET)
+    public ResponseEntity<Object> getDocumentBinaryContentbyDocumentId(
+        @PathVariable("documentId") UUID documentId) {
 
         ResponseEntity documentMetadata = documentManagementService.getDocumentMetadata(documentId);
-        if (documentManagementService.checkUserPermission(documentMetadata, documentId, authorization, Permission.READ)) {
+        if (documentManagementService.checkUserPermission(documentMetadata, documentId, Permission.READ)) {
             return documentManagementService.getDocumentBinaryContent(documentId);
 
         }
-        LOG.error("User doesn't have read permission on requested document {}", HttpStatus.FORBIDDEN);
+        LOG.error("User doesn't have read permission on requested document " + HttpStatus.FORBIDDEN);
```

Comment:
please revert all instances of this + symbols

Reply:
Done.